src/docs/identifiers.md

```diff
@@ -88,15 +92,14 @@ KEGG is actually a set of databases, each with its own prefix, usually of form `
 
 ## Recommended IDs for use within NMDC
 
-The NMDC schema is annotated with the set of IDs that are allowed to act as primary keys for instances of each class.
-
-For example the class [OrthologyGroup](https://microbiomedata.github.io/nmdc-metadata/docs/OrthologyGroup) has a
-description of the IDs allowed on the class web page, the first listed
-is [KEGG.ORTHOLOGY](https://registry.identifiers.org/registry/kegg.orthology)
+The NMDC Schema is annotated with the set of IDs, ordered by preference, that are allowed to act as primary keys 
+for instances of each class. For example the class [OrthologyGroup](https://microbiomedata.github.io/nmdc-metadata/docs/OrthologyGroup) has a description of the IDs allowed 
+on the class web page, the first listed is [KEGG.ORTHOLOGY](https://registry.identifiers.org/registry/kegg.orthology).  The full URL for each is in the jsonld context file, 
+[jsonschema/nmdc.context.jsonld](...).
```

Comment:
I don't think I totally get this yet, but that isn't a reason to hold up the merge.

Reply:
I agree; hard to tell from the diff, but this was text from pre-reorg.  We could do another pass at refining even more of this doc.